lib/ansible/modules/uri.py

```diff
@@ -141,6 +141,11 @@
       - If I(client_cert) contains both the certificate and key, this option is not required.
     type: path
     version_added: '2.4'
+  ca_path:
+    description:
+      - PEM formatted file that contains a CA certificate to be used
```

Comment:
It would be good if you could expand a bit on this, it sounds like it's used for HTTPS cert validation so stating that

* This is used for validation
* It can contain multiple certs (at least I think it should)

Also, why stop at `uri`, this should be added to `get_url` and the `url` lookup plugin to ensure they all offer the same features.

Reply:
I slightly expanded the description in 339fef01ccc8367370b8c5145c2d4ea2cae82403.

I did not test whether `ca_path` can contain multiple certificates. I would only state this in the documentation if we test it. In order to do so, we would need yet another endpoint in the http-test-container. Do you have a use case for this?

Yes, we may add this to other modules as well. For the moment I focus on the uri module.

Reply:
> I did not test whether ca_path can contain multiple certificates. I would only state this in the documentation if we test it. In order to do so

That's fair enough, I was unsure if you had tested it before or not.